Application/Interfaces/IGenericRepository.cs

```diff
@@ -18,6 +13,7 @@ public interface IGenericRepository<T> where T : BaseEntity
     Task<T> DeleteAsync(Guid id);
     Task<T> DeleteSoftAsync(Guid id);
     Task<T> FirstOrDefaultAsync(Expression<Func<T, bool>> predicate);
+    Task<T> FirstOrDefaultTrackingAsync(Expression<Func<T, bool>> predicate);
```

Comment:
Dùng 1 loại thôi, đừng có mix giữa 2 kiểu này

Reply:
không được, as no tracking t ko update enum đc

Reply:
hoặc là bỏ ở cái hàm default, hqua t ko muốn sửa code của m nên t mới viết hàm khác

Reply:
> không được, as no tracking t ko update enum đc

Xin ảnh lỗi hoặc reference nào về issue này để xem thử

Reply:
https://stackoverflow.com/questions/48202403/instance-of-entity-type-cannot-be-tracked-because-another-instance-with-same-key

Reply:
https://stackoverflow.com/questions/12211680/what-difference-does-asnotracking-make

Reply:
> https://stackoverflow.com/questions/48202403/instance-of-entity-type-cannot-be-tracked-because-another-instance-with-same-key
Check lại nhé, cái này không liên quan đến issue m đang nói

Reply:
Rồi cuối cùng là link nào mới là cái issue m nói, m check kỹ đừng có gửi bậy link vào

Reply:
> https://stackoverflow.com/questions/12211680/what-difference-does-asnotracking-make

Cái này cũng liên quan gì tới issue m nói?

Reply:
> > https://stackoverflow.com/questions/12211680/what-difference-does-asnotracking-make
> 
> Cái này cũng liên quan gì tới issue m nói?

ảo ma,t gặp 2 lần r, giờ tracking lại không bị